### PR TITLE
chore(tests): fix tests to align with code refactoring

### PR DESCRIPTION
**Pull request by tinesoft**
- pin version of `@types/jasmine` to `2.5.41` (for compatibility with typescript 2.0)

### CODE REVIEW

File: src/directives/share-button/share-button.directive.ts [hunk 1]
```diff
@@ -9,28 +9,37 @@ import {
 } from '@angular/core';
 
 import { ShareButtonsService } from '../../services/share-buttons.service';
-import { ShareArgs } from '../../helpers';
+import { ShareArgs, ShareProvider, Helper } from '../../helpers';
 
 @Directive({
-    selector: '[shareButton]'
+    selector: '[sbShareButton]'
```
Comment (MurhafSousli) [inline src/directives/share-button/share-button.directive.ts]: There is no need to prefix the directive `[shareButton]`
usage should be something like this:
`<button [shareButton]="'facebook'" [sbTitle]="'HELLO'" [sbShowCount]="true">fb</button>`

File: src/services/share-buttons.service.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Http, Jsonp, Headers, RequestOptions } from '@angular/http';
 
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/operator/catch';
-import 'rxjs/add/Observable/empty';
+import 'rxjs/add/observable/empty';
```
Comment (MurhafSousli) [inline src/services/share-buttons.service.ts]: Nice

File: src/directives/share-button/share-button.directive.ts (hunk 1 above)
Comment (tinesoft) [inline src/directives/share-button/share-button.directive.ts]: yeah  i prefixed it to keep consistency with the other inputs/outputs of the directive

Comment (MurhafSousli) [inline src/directives/share-button/share-button.directive.ts]: @tinesoft no need, just it's input that can conflict with other directives, this one can remains `shareButton`

Comment (tinesoft) [inline src/directives/share-button/share-button.directive.ts]: ok, it doesn't matter to me really...
I'll update and push back

**Pull request merged**